libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -54,6 +55,8 @@ Improvements and New Features
 - The ``std::mismatch`` algorithm has been optimized for integral types, which can lead up to 40x performance
   improvements.
 
+- The ``_LIBCPP_ENABLE_CXX26_REMOVED_STRSTREAM`` macro has been added to make the declarations in ``<strstream>`` available.
```

Comment:
This macro should be documented in `docs/UsingLibcxx.rst` too.

Reply:
Thank you for the reminder. I'll update the other PRs too.